README.md

```diff
@@ -67,6 +67,13 @@ This will create an Azure File Share named `myshare` (if it does not exist)
 and start a Docker container in which you can use `/data` directory to directly
 read/write from cloud file share location using SMB protocol.
 
+You can specify additional volume options to customize the owner, group, and permissions for
+files and directories. See the `mount.cifs(8)` man page more details on these options.
+
+```shell
+$ docker volume create --name rabbitmq -d azurefile -o share=rabbitmq -o uid=999 -o gid=999 -o filemode=0600 -o dirmode=0755
```

Comment:
this is a bit long, maybe separate into new line with `\`

```
$ docker volume create --name rabbitmq -d azurefile \
    -o share=rabbitmq \
    -o uid=999 \
    -o gid=999 \
    -o filemode=0600 \
    -o dirmode=0755
```

and it would be great if you could avoid the rabbitmq name. It looks confusing.